apis/accessing-endpoints.md

```diff
@@ -0,0 +1,42 @@
+# Accessing API Endpoints
+
+Kubecost Cloud has select endpoints available for API access. The default API access endpoint will be `api.app.kubecost.com`. Obtaining access involves generating an API key value, which is then used to generate a personal access token, which is used to retrieve your team ID value. The token and team ID value will then allow you to make queries.
+
+## Access tutorial
+
+### Step 1: Generating an API key value
+
+To get started, visit *Settings* in the Kubecost Cloud UI, then under 'My API Key', select *Generate Key*. A key value will generate in a text box with a lifespan of one month. Save this value, as it will never be retrievable later. Kubecost will not save this value. You can remove the key value, and consequently access to it, by selecting *Remove Key*. This will not provide a warning after being selected, so be careful to not permanently delete your key.

```

Comment:
We _do_ save this key as we need to validate it, so I removed that from the paragraph. While users do need to be careful and store the key once generated (it will not be displayed again), it's not a major issue if they lose access to it. They can remove (ie. revoke) the existing key then create another. 

I've tried to rephrase to convey this message but feel free to refactor if you see fit!


```suggestion
To get started, visit *Settings* in the Kubecost Cloud UI, then under 'My API Key', select *Generate Key*. A key value will generate in a text box with a lifespan of one month. Save this value, as it will not be retrievable later. Only one key can exist at any given time for a given user. Remove the existing key to generate a new one. 

```